GenerateJSCode.php

```diff
@@ -230,12 +230,14 @@ public function generateInitialJSCode($datasource, $options, $dbspecification, $
 
         $this->generateAssignJS(
             "INTERMediatorOnPage.isNativeAuth",
-            (isset($options['authentication']) && isset($options['authentication']['user'])
-                && ($options['authentication']['user'][0] === 'database_native')) ? "true" : "false");
+            ((isset($options['authentication'])
+                && isset($options['authentication']['user'])
+                && ($options['authentication']['user'][0] === 'database_native'))
+                || ((new LDAPAuth())->isActive)) ? "true" : "false");
```

Comment:
Please add 
require_once(dirname(__FILE__) . '/LDAPAuth.php');
in GenerateJSCode.php. It seems to be needed.